.github/workflows/main-ci.yml

```diff
@@ -305,21 +305,31 @@ jobs:
       { name: 'Install Java', uses: actions/setup-java@v1, with: { java-version: '11.0.2' } },
       {
         name: 'Download editor',
+        shell: bash,
```

Comment:
Last time I fixed this, I spent a lot of time trying to figure out how to run the scripts in PowerShell.

It turned out then, that the bash shell worked well, it was just a problem with the encoding of the cert from the b64 Github secret into the cert file.

And, after those fixes, I also forgot to add it in the main-ci.yml script.
This version is used as the base version for each release, and is the one the users are most likely to download from the web.